src/cordova/apple/ios.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,32 +0,0 @@
-{
-  "pins" : [
-    {
-      "identity" : "cocoalumberjack",
-      "kind" : "remoteSourceControl",
-      "location" : "https://github.com/CocoaLumberjack/CocoaLumberjack",
-      "state" : {
-        "revision" : "0188d31089b5881a269e01777be74c7316924346",
-        "version" : "3.8.0"
```

Comment:
This is probably what broke in https://github.com/Jigsaw-Code/outline-client/issues/1800. We should be using 3.7 with iOS 11.